legal-api/src/legal_api/services/authz.py

```diff
@@ -170,29 +170,32 @@ def get_allowable_filings_dict():
                     'regular': {
                         'legalTypes': ['BEN', 'BC', 'ULC', 'CC', 'C', 'CBEN', 'CUL', 'CCC'],
                         'blockerChecks': {
-                            'business': [BusinessBlocker.BUSINESS_FROZEN,
+                            'business': [BusinessBlocker.DEFAULT,
                                          BusinessBlocker.NOT_IN_GOOD_STANDING,
-                                         BusinessBlocker.IN_DISSOLUTION],
+                                         BusinessBlocker.IN_DISSOLUTION,
+                                         BusinessBlocker.DRAFT_PENDING],
```

Comment:
No need to add `DRAFT_PENDING`. DEFAULT covers both DRAFT_PENDING and BUSINESS_FROZEN.

Ref. 
https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/authz.py#L659
https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/authz.py#L663